src/pages/workspace/WorkspaceInitialPage.tsx

```diff
@@ -156,6 +164,7 @@ function WorkspaceInitialPage({policyDraft, policy: policyProp, policyMembers, r
             brickRoadIndicator: hasGeneralSettingsError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
             routeName: SCREENS.WORKSPACE.PROFILE,
         },
+        ...(isCollectPolicy ? collectPolicyMenuItems : []),
```

Comment:
We also want to only show this page to admins, no?

Reply:
See comments here:
https://github.com/Expensify/App/pull/36345#discussion_r1492307470 and https://github.com/Expensify/App/issues/36606

Reply:
Yes I've modified it to only show this page to admins. 

I'll resolve this comment once I've handled the deeplinking aspect. Thanks